app/page.tsx

```diff
@@ -425,3 +435,71 @@ export default function HomePage() {
     </div>
   );
 }
+
+const MarketHighlight = () => {
+  const marketIdList = [
+    '0x8943a769cb0503f7bc4e92a777fcd1aa9decfc33',
+    '0x0539590c0cf0d929e3f40b290fda04b9b4a8cf68',
+  ];
```

Comment:
we should have a place for this outside, like a file or something, so we to update it we will only need update that file, making updates on highlights very clear.